frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx

```diff
@@ -105,7 +105,9 @@ export const PopinFixedHeader = (props) => {
   const filteredHeaderButtons = headerButtons ? headerButtons.filter(action => action.showAction) : []
 
   return (
-    <div className={classnames('wsContentGeneric__header', `${customClass}__header`)}>
+    <div className={editTitle
+      ? classnames(classnames('wsContentGeneric__header', `${customClass}__header__isEditing`), 'wsContentGeneric__header__isEditing') : classnames('wsContentGeneric__header', `${customClass}__header`)}
+    >
```

Comment:
this can be simplified

write it with using only one call to classnames() and by writing each classes only once

Reply:
done